crates/interpreter/src/interpreter.rs

```diff
@@ -53,31 +53,6 @@ pub struct Interpreter {
     pub next_action: Option<InterpreterAction>,
 }
 
-#[derive(Debug, Clone)]
-pub struct InterpreterResult {
-    pub result: InstructionResult,
-    pub output: Bytes,
-    pub gas: Gas,
-}
-
-#[derive(Debug, Clone)]
-pub enum InterpreterAction {
-    SubCall {
-        /// Call inputs
-        inputs: Box<CallInputs>,
-        /// The offset into `self.memory` of the return data.
-        ///
-        /// This value must be ignored if `self.return_len` is 0.
-        return_memory_offset: Range<usize>,
-    },
-    Create {
-        inputs: Box<CreateInputs>,
-    },
-    Return {
-        result: InterpreterResult,
-    },
-}
```

Comment:
Can you revert this? I liked more having structs first than impls

Reply:
sure, but I very much dislike this style:

```rust
struct A;
struct B;

impl A {}
impl B {}
```

imo this makes it a lot harder to navigate

Reply:
reverted